### DISCUSSION TITLE
PR or ETA for `signTransaction` ?

### DISCUSSION THREAD

**Discussion by Gldywn** [wevm/viem#713] [Question]
Hello @jxom @tmm 👋 

We are currently using viem in our project and we need to use the wallet action `signTransaction`.
Do you have any ETA on the release date for this particular function, if not are you open to PRs for this one.

Thanks a lot 👍

**Comment by jxom** [comment]
If you need to do this for a Private Key wallet, you can do this already! The returned account on `privateKeyToAccount` has a `signTransaction` function.

**Reply by Gldywn** [reply]
Unfortunately not, we use the `WalletClient` exposed by **Wagmi**, which itself internally uses viem (and expose the available methods).
